src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Conformance/ConformanceBuilderTests.cs

```diff
@@ -88,6 +88,8 @@ public void GivenAConformanceBuilder_WhenAddingDefaultInteractions_ThenAuditEven
             Assert.False(noDelete);
         }
 
+// STU3 doesn't allow empty interactions so we had have Parameters in capability statement.
```

Comment:
Do you mind elaborating on what's happening here? The test is that Parameters is not in the capability statement

Reply:
![image](https://user-images.githubusercontent.com/1523833/118019029-5f50a700-b30d-11eb-8aa7-98c15e76f16e.png)

http://hl7.org/fhir/stu3/capabilitystatement.html

this is what we returned before changes:
```
  "type": "Parameters",
                    "profile": {
                        "reference": "http://hl7.org/fhir/StructureDefinition/Parameters"
                    },
                    "versioning": "versioned",
                    "conditionalCreate": false,
                    "conditionalUpdate": false
```
and it trigger validation error.
So I add delete interaction for Parameters. That sound like less invasive options to solve issue with validator. You can't create or update parameter, and it wouldn't complain about delete, but there is nothing to delete anyway.

Reply:
Ah ok. Should it even be in the CapabilityStatement at all? I don't think it makes sense since it isn't stored